src/Middleware.php

```diff
@@ -49,22 +49,24 @@ static function (ResponseInterface $response) use ($cookieJar, $request): Respon
      * Middleware that throws exceptions for 4xx or 5xx responses when the
      * "http_error" request option is set to true.
      *
+     * @param int|null $truncateBodyAt The length of the body to include in an exception message before truncation.
+     *
      * @return callable(callable): callable Returns a function that accepts the next handler.
      */
-    public static function httpErrors(): callable
+    public static function httpErrors(int $truncateBodyAt = null): callable
```

Comment:
Let's use our `BodySummarizerInterface` interface here too.

Reply:
Yeh, that's fair enough now that we have the class that implements this.